From 7db1109c841043af4a027b23e62645172f4879ca Mon Sep 17 00:00:00 2001
From: Mihails Strasuns <mihails.strasuns.contractor@sociomantic.com>
Date: Thu, 19 May 2016 12:50:31 +0300
Subject: [PATCH 06/17] Keep -release flag in runtime

Not doing so results in major performance penalty in ported
applications. Unfortunate side effect is that stomping prevention
assert will now segfault binaries instead of priting stack trace.
It still can be checked in gdb though as debug symbols are being kept.
---
 posix.mak | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/posix.mak b/posix.mak
index 77a9de14..700ab7a5 100644
--- a/posix.mak
+++ b/posix.mak
@@ -60,7 +60,7 @@ ifeq ($(BUILD),debug)
 	UDFLAGS += -g -debug -debug=CheckStompingPrevention
 	DFLAGS:=$(UDFLAGS)
 else
-	UDFLAGS += -O -g -debug=CheckStompingPrevention
+	UDFLAGS += -O -release -g -debug=CheckStompingPrevention
 	DFLAGS:=$(UDFLAGS) -inline # unittests don't compile with -inline
 endif
 
-- 
2.11.0

